client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js

```diff
@@ -125,8 +127,31 @@ export default class {
 
   /**
    *  @ngdoc method
-   *  @name managerApp.service:tucVoipServiceAlias#fetchRedirectNumber
-   *  @methodOf managerApp.service:tucVoipServiceAlias
+   *  @name managerApp.service:voipServiceAlias#fetchNumberSounds
+   *  @methodOf managerApp.service:voipServiceAlias
+   *
+   *  @description
+   *  <p>Fetch sounds available for a given number.</p>
+   *
+   *  @param  {VoipService} number (destructured) The given VoipService number.
+   */
+  fetchNumberSounds({ billingAccount, serviceName }) {
+    return this.OvhApiTelephony.EasyHunting().Sound().v6().query({
+      billingAccount,
+      serviceName,
+    }).$promise.then(soundIds => this.$q.all(
+      soundIds.map(soundId => this.OvhApiTelephony.EasyHunting().Sound().v6().get({
+        billingAccount,
+        serviceName,
+        soundId,
+      })),
```

Comment:
I bet it was more something like this:
```suggestion
      }).$promise),
```
And `.$promise` L148 could be deleted.